src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/Stats.java

```diff
@@ -29,6 +30,7 @@
   private Map<Integer, ConnectedComponent> graphs;
 
   private Set<String> mutedGraphNodes;
+  private Set<String> mutedActions;
```

Comment:
It would be nice if we don't add code to this class anymore and in future, we just deprecate it. Adding anything to this class restricts us to have a uniform view of things across all virtualInstances during RCA-IT and we won't be able to test things like nodes running different versions of graphs and different muted rca as such.

we have this new kid in the block to pass around things - `appContext`. Any chance we can use it ?

Reply:
yup, using it now.